src/NaadSocketClient.php

```diff
@@ -301,6 +314,100 @@ protected function isHeartbeat( SimpleXMLElement $xml ): bool
         return !empty($sender);
     }
 
+    /**
+     * Finds any alerts that were missed by the socket connection.
+     *
+     * @param SimpleXmlElement $xml Heartbeat XML.
+     *
+     * @return array An array containing the heartbeat references for any alerts
+     *               that are not already in the database.
+     */
+    protected function findMissedAlerts(SimpleXMLElement $xml): array
+    {
+        $rawReferences = explode(' ', $xml->references);
+        $references = [];
+
+        // Separate the references value into sender, id, and sent parts.
+        foreach ($rawReferences as $reference) {
```

Comment:
```suggestion
        foreach ($rawReferences as $rawReference) {
```
this might be a better name, as this might fit better with the foreach ... it seems to close to $references = []

Reply:
Made this change.